hysds_commons/elasticsearch_utils.py

```diff
@@ -210,18 +210,19 @@ def update_document(self, index, _id, body, refresh=False):
                 self.logger.exception(e)
             raise ElasticsearchException(e)
 
-    def delete_by_query(self, index, query, **kwargs):
+    def delete_by_query(self, index, **kwargs):
         """
+        https://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch.Elasticsearch.delete_by_query
         Deletes documents matching the provided query.
         :param index: str, Elasticsearch index
-        :param query: dict, query
+        param query: dict, query
```

Comment:
@DustinKLo: should we delete this entirely?

Reply:
`hysds_commons` isn't yet merged to `develop-es7` so this was the only way to test my lightweight-jobs
will change after changes to `hysds_commons` gets approved

Reply:
good point, if anything users can look into the elasticsearch-py documentation if they need to use the `delete_by_query` function
ill remove the comment